app/models/forms/author.rb

```diff
@@ -0,0 +1,7 @@
+module Forms
+  class Author < Contributor
```

Comment:
The only difference between the form object for an author and a contributor, is that an author form object creates author active records and a contributor creates contributor active records.